modules/proxistoreBidAdapter.js

```diff
@@ -59,17 +72,28 @@ function _createBidResponse(response) {
     vastUrl: response.vastUrl,
     vastXml: response.vastXml,
     dealId: response.dealId
-  }
+  };
 }
-
 /**
  * Determines whether or not the given bid request is valid.
  *
  * @param bid  The bid params to validate.
  * @return boolean True if this is a valid bid, and false otherwise.
  */
+
 function isBidRequestValid(bid) {
-  return !!(bid.params.website && bid.params.language);
+  const hasNoAd = function() {
+    const pxNoAds = storage.getDataFromLocalStorage(`PX_NoAds_${bid.params.website}`);
+    if (!pxNoAds) {
+      return false;
+    } else {
+      const storedDate = new Date(pxNoAds);
+      const now = new Date();
+      const diff = Math.abs(storedDate.getTime() - now.getTime()) / 60000;
+      return diff <= 5;
+    }
+  }
+  return !!(bid.params.website && bid.params.language) && !hasNoAd();
```

Comment:
You don't have to change this one, this is small suggestion from my side: you could remove double negation here.  
You check `!hasNoAd()`, you could rename function to `hasAd()` and invert logic in that function and then just check `hasAd()`. To me, it's more readable.